src/edu/macalester/comp127/hw0/Area.java

```diff
@@ -21,7 +21,7 @@ public static double getSquareArea(double sideLength) {
      */
     public static double getCircleArea(double radius) {
         // TODO: replace me with the correct formula
-        return radius * 1.0;
+        return radius * radius * 3.1415926;

```

Comment:
For future reference, you can also use Math.pow(radius, 2) and Math.PI to perform these calculations more precisely!